src/index.html

```diff
@@ -22,6 +22,370 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header__navigation navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a
+              class="navigation__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="navigation__item"
+            data-qa="nav-hover"
+          >
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        data-qa="card"
+        class="card"
+      >
+        <img
+          src="./images/imac.jpeg"
```

Comment:
The src attribute should use a relative path, not a path that looks like it's absolute. Use './images/imac.jpeg' instead of '/src/images/imac.jpeg'.